src/main/resources/swagger/thurloe.yaml

```diff
@@ -22,77 +22,67 @@ paths:
       operationId: query
       summary: Query the database using a custom subset of userId, key and value
       produces:
-        - text/plain
         - application/json
+        - text/plain
       parameters:
         - name: X-FireCloud-Id
           in: header
           description: FireCloud ID Token to be passed as a header
           required: true
           type: string
         - name: userId
-          description: User ID of the owner of the key/value pair
+          description: User ID of the owner of the key-value pair
           required: false
           type: array
           items:
             type: string
           collectionFormat: multi
           in: query
-          description: >
-            Show only (User,(Key,Value)) items which are assigned to this user ID
         - name: key
-          description: Key of the key/value pair
+          description: Key of the key-value pair
           required: false
           type: array
           items:
             type: string
           collectionFormat: multi
           in: query
-          description: >
-            Show only (User,(Key,Value)) items which have this key
         - name: value
-          description: Value of the key/value pair
+          description: Value of the key-value pair
           required: false
           type: array
           items:
             type: string
           collectionFormat: multi
           in: query
-          description: >
-            Show only (User,(Key,Value)) items which have this value
       tags:
-        - Key/value pairs
+        - Key-value pairs
       responses:
         200:
           description: OK
           schema:
             type: array
             items:
-              type: "object"
+              type: object
           examples:
-            application/json: |-
-              [{
-                "userId": "string",
-                "keyValuePair": {
-                  "key": "string",
-                  "value": "string"
-                }
-              }]
+            application/json:
+              -
+                userId: 123
+                keyValuePair:
+                  key: foo
+                  value: bar
```

Comment:
Nice job switching over from json - devops' scans should be happier if they ever check Thurloe's swagger.